inst/rmarkdown/templates/assignment_validation/skeleton.Rmd

```diff
@@ -0,0 +1,62 @@
+---
+title: "Assignment Validation"
+author: "WSP | Parsons Brinckerhoff"
+output: html_document
+---
+
+
+```{r setup}
+library(outviz)
+library(dplyr)
+library(knitr)
+
+
+```
+
+```{r area_type}
+volume_var <- "volume"
+count_var <- "count"
+links <- links %>%
+  mutate(area_type = factor(
+    area_type,  labels = c("Urban", "Exurban", "Suburban", "Rural"))
+  )
+```
+
+
+```{r validation}
+plot_validation(links, volume_var, count_var)
+plot_validation(links, volume_var, count_var, color_field = "facility_group")
+```
+
+```{r mdd}
+plot_mdd(links, volume_var, count_var)
+plot_mdd(links, volume_var, count_var, color_field = "facility_group")
+```
+
+```{r rmse}
+table_rmse(links, volume_var, count_var, group_field = "facility_group") %>%
+  mutate(`Target` = c(25, 50, 50, 65, 40)) %>%
+  kable(digits = 2, caption = "RMSE by Facility Group")
+
+table_rmse(links, volume_var, count_var, group_field = volume_var) %>%
+  mutate(`Target` = c(120, 45, 35, 30, 40)) %>%
+  kable(digits = 2, caption = "RMSE by Volume Class")
+
+table_rmse(links, volume_var, count_var, group_field = "area_type") %>%
+  kable(digits = 2, caption = "RMSE by Area Type")
+```
+
+```{r flows}
+table_flow(links, volume_var, count_var, group_field = "facility_group") %>%
```

Comment:
@gregmacfarlane 
When I try to knit this file, I get the following error message:
![error](https://cloud.githubusercontent.com/assets/10221750/13878246/6b289c56-ece6-11e5-8201-32f791cc284e.png)